examples/reports/html-reporter-template-data.json

```diff
@@ -79,39 +83,51 @@
           },
           {
             "key": "Date",
-            "value": "Thu, 25 Aug 2016 14:22:01 GMT"
+            "value": "Tue, 22 Nov 2016 20:20:50 GMT"
           },
           {
             "key": "ETag",
-            "value": "W/\"137-PVrRGQ6/XWPrd95mL9VLkA\""
+            "value": "W/\"137-q0lxUUvKUot4wiPUo3uKCg\""
           },
           {
             "key": "Server",
             "value": "nginx/1.8.1"
           },
           {
             "key": "set-cookie",
-            "value": "sails.sid=s%3AKWkP8x3Zi5Dw8iEGxf4rtsVeadIVITil.kefDILpoX6Yda98kJa7ksUS8M5y1tUnu4S7yjKZN5Dg; Path=/; HttpOnly"
+            "value": "sails.sid=s%3AW0Ng8epFgCZJpDdMo_odMqH3BbWD38R9.90SxQF2ceaA7UoFSfYDoZnjOhENQDYmyV8Bun6s11OM; Path=/; HttpOnly"
           },
           {
             "key": "Vary",
             "value": "Accept-Encoding"
           },
           {
             "key": "Content-Length",
-            "value": "203"
+            "value": "204"
           },
           {
             "key": "Connection",
             "value": "keep-alive"
           }
         ],
-        "body": "{\"args\":{\"source\":\"newman-sample-github-collection\"},\"headers\":{\"host\":\"echo.getpostman.com\",\"accept\":\"*/*\",\"accept-encoding\":\"gzip, deflate\",\"user-agent\":\"PostmanRuntime/2.4.3\",\"x-forwarded-port\":\"443\",\"x-forwarded-proto\":\"https\"},\"url\":\"https://echo.getpostman.com/get?source=newman-sample-github-collection\"}",
-        "responseTime": 1965,
-        "responseSize": 311
+        "body": "{\"args\":{\"source\":\"newman-sample-github-collection\"},\"headers\":{\"host\":\"echo.getpostman.com\",\"accept\":\"*/*\",\"accept-encoding\":\"gzip, deflate\",\"user-agent\":\"PostmanRuntime/3.0.4\",\"x-forwarded-port\":\"443\",\"x-forwarded-proto\":\"https\"},\"url\":\"https://echo.getpostman.com/get?source=newman-sample-github-collection\"}",
+        "cookie": [],
+        "responseTime": 1681,
+        "responseSize": 311,
+        "update": {},
+        "reason": {},
```

Comment:
@czardoz This has been present in the provided data since the HTML reporter was added. I'll remove it.